app/Models/Equipment/Breakage.php

```diff
@@ -57,4 +57,22 @@ public function user()
     {
         return $this->belongsTo('App\Models\Users\User');
     }
+
+    public function saveImage($image)
+    {
+        // save image to /resources/breakages
+        $filename = uniqid() . '.' . $image->extension();
```

Comment:
I would recommend using the image's ID here - it will make it much easier to trace in the filesystem, compared to a random name. Plus, you then don't need to explicitly save this in the DB.